modules/hostap/src/supp_api.c

```diff
@@ -358,13 +358,27 @@ static int wpas_add_and_config_network(struct wpa_supplicant *wpa_s,
 		} else if (params->security == WIFI_SECURITY_TYPE_PSK ||
 				   params->security == WIFI_SECURITY_TYPE_WPA_PSK) {
 			_wpa_cli_cmd_v("set_network %d psk \"%s\"",
-			resp.network_id, params->psk);
+					resp.network_id, params->psk);
```

Comment:
please move formatting changes to a separate commit

Reply:
These were inadvertently pushed. Removed these changes.